### ISSUE TITLE
Alignment detection not working for more than 2 columns

### ISSUE THREAD

**Issue by jabenninghoff** [r-lib/styler#943]
According to https://styler.r-lib.org/articles/detect-alignment.html, styler detects both right-aligned and left-aligned columns for function calls like `tibble::tribble()`.

While two-column tribble calls are not changed (alignment is detected):

```r
  tibble::tribble(
    ~x,    ~y,
    "one", TRUE,
    "two", FALSE
  )
```

Three (or more) column tribble calls are restyled. This left-aligned example:

```r
  tibble::tribble(
    ~x,    ~y,    ~z,
    "one", TRUE,  1L,
    "two", FALSE, 2L
  )
```

is unexpectedly restyled to:

```r
  tibble::tribble(
    ~x, ~y, ~z,
    "one", TRUE, 1L,
    "two", FALSE, 2L
  )
```

**Comment by lorenzwalthert** [comment]
@jabenninghoff thanks for the bug report. Indeed, left-aligned calls with more than two columns are a false negative. Can you tell me if the problem (probably you had some more complex code inititally on which you want to try it out) persists with #945? I added your reprex as a unit test.

**Comment by jabenninghoff** [comment]
Thanks! The more complex example is [here](https://github.com/jabenninghoff/rdev/blob/24edc8460ef106c806c693580fc504a8065ba551/R/setup.R#L447-L463), four-column output. Tested with #945, this fixes the issue. Thanks for the quick response!
